libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -141,17 +141,17 @@ typedef void (^SFSnapshotViewControllerDismissalBlock)(UIViewController* snapsho
 /**
  The Connected App ID configured for this application.
  */
-@property (nonatomic, copy, nullable) NSString *connectedAppId;
+@property (nonatomic, copy, nullable) NSString *connectedAppId SFSDK_DEPRECATED(6.0, 7.0, "Use appConfig.remoteAccessConsumerKey to specify the consumer key.");
```

Comment:
Could we remove these in `6.0`?

Reply:
We could, I suppose.  We never deprecated them though.  Happy to do whatever the team wants.

Reply:
Makes sense.